src/ocean/net/device/Berkeley.d

```diff
@@ -1353,7 +1355,14 @@ deprecated public class IPv4Address : Address
         static uint parse(istring addr)
         {
                 char[64] tmp;
-                return ntohl(inet_addr(toStringz(addr, tmp)));
+                if ((addr.length + 1) > tmp.length)
```

Comment:
You know what I mean? It doesn't seem very cool to me to say to users: "This function is deprecated. Instead, please just copy the bits of its internal logic that you need into your application code." (Where some of the internal logic is more than just a simple matter of `~ '\0'`.)